sources/api/schnauzer/src/helpers.rs

```diff
@@ -1854,35 +2028,109 @@ mod test_add_unresolvable_hostname {
         T: Serialize,
     {
         let mut registry = Handlebars::new();
-        registry.register_helper(
-            "add_unresolvable_hostname",
-            Box::new(add_unresolvable_hostname),
-        );
+        registry.register_helper("localhost_aliases", Box::new(localhost_aliases));
+        registry.register_helper("etc_hosts_entries", Box::new(etc_hosts_entries));
 
         registry.render_template(tmpl, data)
     }
 
     #[test]
     fn resolves_to_localhost_renders_entries() {
+        // Given a configured hostname that does not resolve in DNS,
+        // When /etc/hosts is rendered,
+        // Then an additional alias shall be rendered pointing the configured hostname to localhost.
         let result = setup_and_render_template(
-            "{{add_unresolvable_hostname name}}",
-            &json!({"name": "localhost"}),
+            r#"{{localhost_aliases "ipv4" hostname hosts}}"#,
+            &json!({"hostname": "localhost"}),
+        )
+        .unwrap();
+        assert_eq!(result, "localhost")
+    }
+
+    #[test]
+    fn hostname_resolves_to_static_mapping() {
+        // Given a configured hostname that does not resolve in DNS
+        // and an /etc/hosts configuration which points that contains that hostname as an alias to an IP address,
```

Comment:
Nit: I think there are extra words here? Should the line be:
```suggestion
        // and an /etc/hosts configuration that contains that hostname as an alias to an IP address,
```